src/context/github/GithubState.tsx

```diff
@@ -143,6 +143,7 @@ const GithubState: FunctionComponent<Props> = (props) => {
       const data = await response.json();
       console.log(data);
       typeof data === 'string' && toast.error(data);
+      localStorage.deleteItem('oauthCode')
```

Comment:
Why are you deleting the oauthCode from localStorage here?

Reply:
I thought of deleting token incase of authentication error so that the user can authorize their account again. I should probably remove this line.